app/assets/javascripts/views/schedule.js

```diff
@@ -53,6 +54,24 @@ Yacs.views.schedule = function (data) {
     crnListElement.textContent = 'CRNs: ' + scheduleData.crns.join(', ');
   }
 
+  /* this is before `if(data.schedules.length==0)` because clear selection should */
```

Comment:
Should I not document that the handler is placed before the no schedule to show branching point, or make it clear some other way?